src/ACEtk/block/LevelScatteringDistribution.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef NJOY_ACETK_BLOCK_LEVELSCATTERING
+#define NJOY_ACETK_BLOCK_LEVELSCATTERING
+
+// system includes
+
+// other includes
+#include "ACEtk/block/details/Base.hpp"
+
+namespace njoy {
+namespace ACEtk {
+namespace block {
+
+/**
+ *  @class
+ *  @brief The energy distribution uses analytical level scattering
+ *
+ *  The LevelScatteringDistribution class contains two parameter values that
+ *  determine the level scattering distribution (in the centre of mass):
+ *    Eprime = C2 * ( E - C1 )
+ *  with:
+ *    C1 = ( A + 1 ) / A * abs( Q )
+ *    C2 = ( A / ( A + 1 ) )^2
+ *
+ *  The content of the XSS array are only C1 and C2
+ */
+class LevelScatteringDistribution : protected details::Base {
+
+  /* fields */
+  double emin_;
+  double emax_;
+
+  /* auxiliary functions */
+  #include "ACEtk/block/LevelScatteringDistribution/src/verifySize.hpp"
+
+public:
+
+  /* constructor */
+  #include "ACEtk/block/LevelScatteringDistribution/src/ctor.hpp"
+
+  /**
+   *  @brief Return the minimum incident energy for the distribution
+   */
+  double minimumIncidentEnergy() const { return this->emin_; }
```

Comment:
Are `emin` and `emax` specified ACE speak, justifying additional interface?  Or do these not have an official ACE name?

Reply:
The minimum and maximum energy are not part of the data for the law (there's only C1 and C2 that are needed). However, these energy values need to be given so that we can generate the obligatory probability = 1 when the law is the only one given. It's part of the abstraction that allows us to omit that probability when serving up data.

As such, they do not have an official ACE name, but we could give them one to make it consistent and that might be the better choice. All other DLW components (see feature/dlw-components-too) have the same interface functions as well.